production/helm/loki/Chart.yaml

```diff
@@ -4,7 +4,7 @@ name: loki
 description: Helm chart for Grafana Loki in simple, scalable mode
 type: application
 appVersion: 2.6.1
-version: 3.3.4
+version: 3.4.0
```

Comment:
Would 3.3.5 suffice? I'm not sure about the impact.

Reply:
I personally have no strong opinion on this
But since it was like a "new feature" ( from the helm chart point of view) I bumped the minor version, even if the change is really minor